frontend/src/Components/LoginPage/LoginPage.module.css

```diff
@@ -89,3 +83,41 @@
   grid-column: 1 / 3;
   justify-self: center;
 }
+
+.sessionContainer {
+  display: grid;
+  gap: 1ch;
+  grid-template-columns: 1fr min-content;
+  grid-column: 2 / 3;
+  align-self: center;
+  align-items: center;
+}
+
+.sessionLink {
+  color: var(--primary);
+  min-width: 19ch;
+}
+
+.refreshButton {
+  color: var(--primary);
+  fill: var(--primary);
+  border: unset;
+  border: 1px solid var(--primary);
```

Comment:
Wouldn't the second border value just override the first value, making the first value unnecessary?